build.gradle

```diff
@@ -770,33 +775,6 @@ project(':iceberg-orc') {
   }
 }
 
-project(':iceberg-parquet') {
```

Comment:
Removed the module as keeping empty jar to support compatibility was unnecessary. Users who depend on `iceberg-parquet` would obviously depend on `iceberg-core`. So, no impact for class imports.

But the Users build files might have to update to remove `iceberg-parquet ` dependency. That had to be done anyways whether in this version or other when we delete it. We can update release notes about this.